src/app/OrderBook.tsx

```diff
@@ -6,25 +6,59 @@ export function OrderBook({ orders }: { orders: OrderbookLine[] }) {
   //returns simple orderbook of buys/sells
   const adexState = useContext(AdexStateContext);
   const { buys, sells } = adexState.currentPairOrderbook;
+  // return (
```

Comment:
I think we can go ahead and delete unused code, no need to keep it in comments - just delete it, putting it in comments just clutters everything.